docs/UserGuide.md

```diff
@@ -1,226 +1,23 @@
----
-layout: page
-title: User Guide
----
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+### Format: ```add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [d/DOB]```
```

Comment:
before ###Format, we'd need a line __Optional Parameter 2: Date of Birth__ for consistency with Qiao Ran